apps/extension/webpack/webpack.common.js

```diff
@@ -115,8 +115,6 @@ const config = (env) => ({
   plugins: [
     new webpack.DefinePlugin({
       // passthroughs from the environment
-      "process.env.EXTENSION_PREFIX": JSON.stringify(""), // this env var MUST be set, however if it has a value the keyring will break
-      "process.env.PORT_PREFIX": JSON.stringify(process.env.PORT_PREFIX || "talisman"),
```

Comment:
These never did anything. The `DefinePlugin` will replace the literal code `process.env.EXTENSION_PREFIX` with `""`, but this doesn't exist anywhere!

Instead, pjs deps do a process.env["PORT_PREFIX"]